jwt/src/main/java/io/scalecube/security/acl/AccessControlListJwtExample.java

```diff
@@ -0,0 +1,71 @@
+package io.scalecube.security.acl;
+
+import io.jsonwebtoken.Jwts;
+import io.scalecube.security.DefaultJwtAuthenticator;
+import io.scalecube.security.JwtKeyResolver;
+import io.scalecube.security.auth.AccessControl;
+import io.scalecube.security.auth.Authenticator;
+import io.scalecube.security.auth.Authorizer;
+import io.scalecube.security.auth.BaseAccessControl;
+import java.security.Key;
+import java.security.NoSuchAlgorithmException;
+import javax.crypto.KeyGenerator;
+
+public class AccessControlListJwtExample {
+
+  private static final String OWNER = "owner";
+  private static final String ADMIN = "admin";
+  private static final String MEMBER = "member";
+
+  /**
+   * an example.
+   *
+   * @param args ignored
+   * @throws NoSuchAlgorithmException when HmacSHA256 is not supported
+   */
+  public static void main(String[] args) throws NoSuchAlgorithmException {
+
+    Authorizer permissions =
+        PermissionsAuthorizer.builder()
+            .permission("repo.create", OWNER, ADMIN)
```

Comment:
i suggest to rename this method 'premissions' to 'allow'

Reply:
The term grant is often used in this context.

Reply:
+1 to grant then